test/spec/support/container_helper.rb

```diff
@@ -0,0 +1,20 @@
+module ContainerHelper
+
+  # Checks if a deployed container exists with given name
+  # @param [String] name of the container to check
+  def container_exist?(name)
+    k = run "kontena container ls"
```

Comment:
We just got `kontena container ls -q`, this would be a good use for it :)

Ideally this would also match on the `/vpn.server-1` suffix or something, to avoid false-positives with other similarly named containers, but that's not strictly necessary for this  PR.